Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -121,6 +121,17 @@ public void OpenDCCTool(UnityEngine.Object asset)
             }
         }
 
+        public void CloseDCCTool() {
```

Comment:
why do we need this?

Reply:
We used to automatically close the DCC tool when the asset is opened. This logic was removed and that breaks the regenerator.